src/command_modules/azure-cli-profile/azure/cli/command_modules/profile/commands.py

```diff
@@ -4,13 +4,24 @@
 # --------------------------------------------------------------------------------------------
 
 #pylint: disable=line-too-long
+from collections import OrderedDict
 
 from azure.cli.core.commands import cli_command
 
 cli_command(__name__, 'login', 'azure.cli.command_modules.profile.custom#login')
 cli_command(__name__, 'logout', 'azure.cli.command_modules.profile.custom#logout')
 
-cli_command(__name__, 'account list', 'azure.cli.command_modules.profile.custom#list_subscriptions')
+def transform_account_list(result):
+    transformed = []
+    for r in result:
+        res = OrderedDict([('Name', r['name']), ('CloudName', r['cloudName']), \
+            ('SubscriptionId', r['id']), ('State', r['state']), ('IsDefault', r['isDefault'])])
+        transformed.append(res)
+    return transformed
```

Comment:
Part of me was thinking if the entire body is accomplished by single list generator that will be awesome. And another part of me knows that will be a death trap to the code readers. 

So, Approved.